### ISSUE TITLE
VnV Feedback #1

### ISSUE THREAD

**Issue by V-AS** [Yuanqi-X/Re-ProtGNN#9]
On page 7, under 'Test for Training Loss Optimization,' the input may be a training dataset instead of the full MUTAG dataset.
